config/zones.json

```diff
@@ -4894,7 +4894,8 @@
       "https://github.com/systemcatch",
       "https://github.com/7hibault",
       "https://github.com/nessie2013",
-      "https://github.com/q--"
+      "https://github.com/q--",
+      "https://github.com/IV1T3"
     ],
     "parsers": {
       "production": "PA.fetch_production"
```

Comment:
You should also add the consumption parser here: `"consumption": "PA.fetch_consumption"` :)